app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,23 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <div class="container-fluid">
+    <%= link_to "RecipeApp", root_path, class: "navbar-brand" %>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav ms-auto">
+        <li class="nav-item">
+          <%= link_to "Foods", foods_path, class: "nav-link" %>
+        </li>
+        <li class="nav-item">
+          <%= link_to "Recipes", recipes_path, class: "nav-link" %>
+        </li>
+      </ul>
+    </div>
+    <div>
+      <% if user_signed_in? %>
+        <%= button_to "Logout", destroy_user_session_path, method: :delete, class: "btn btn-outline-secondary", id: "destroy-session" %>
+        <% else %>
+        <%= link_to "Login", new_user_session_path, class: "btn btn-outline-secondary" %>
+        <%= link_to "Register", new_user_registration_path, class: "btn btn-outline-secondary" %>
+      <% end %>
+    </div>
+  </div>
+</nav>
```

Comment:
To apply the bootstrap styling, you can use the CDN links in the application.html.erb to make it work 
for more information - check out the bootstrap [docs](https://getbootstrap.com/docs/5.3/getting-started/introduction/)